packages/unison-gui/unison-gui.2.53.7/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "unison-hackers@lists.seas.upenn.edu"
+authors: [
+  "Trevor Jim"
+  "Benjamin C. Pierce"
+  "Jérôme Vouillon"
+]
+license: "GPL-3.0-or-later"
+homepage: "https://www.cis.upenn.edu/~bcpierce/unison/"
+bug-reports: "https://github.com/bcpierce00/unison/issues"
+dev-repo: "git://github.com/bcpierce00/unison.git"
```

Comment:
I think something is off, this package does not have `build` and `install` instructions, right?

Reply:
Indeed: it simply has an additional dependency on lablgtk. The makefile of the main package decides to build the GUI depending on whether lablgtk is alrady installed.

Reply:
That's the reason why lablgtk is a depopt of unison

Reply:
Oh I see, so that indeed forces a recompilation. Thanks!